lib/vmdb/plugins/asset_path.rb

```diff
@@ -7,6 +7,8 @@ class AssetPath
       attr_reader :name
       attr_reader :path
       attr_reader :namespace
+      attr_reader :node_modules
+      attr_reader :is_gem
```

Comment:
Prefer `gem?` over `is_gem`

Reply:
That means I need to write a method, but sure :)